frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -1,33 +1,30 @@
-import { LemonDivider } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { SupportForm, SupportFormButtons } from 'lib/components/Support/SupportForm'
 import { supportLogic } from 'lib/components/Support/supportLogic'
-import { useEffect } from 'react'
 
 import { SidePanelTab } from '~/types'
 
+import { SidePanelPaneHeader } from '../components/SidePanelPane'
 import { sidePanelStateLogic } from '../sidePanelStateLogic'
 
 export const SidePanelSupport = (): JSX.Element => {
     const { closeSidePanel } = useActions(sidePanelStateLogic)
-    const { selectedTab } = useValues(sidePanelStateLogic)
 
     const theLogic = supportLogic({ onClose: () => closeSidePanel(SidePanelTab.Support) })
     const { title } = useValues(theLogic)
     const { closeSupportForm } = useActions(theLogic)
 
-    useEffect(() => {
-        return () => closeSupportForm()
-    }, [selectedTab])
-
     return (
-        <div className="p-3 max-w-160 w-full mx-auto">
-            <h2 className="text-lg font-bold mb-2">{title}</h2>
-            <LemonDivider />
-            <SupportForm />
-            <div className="flex items-center justify-end gap-2 mt-4">
-                <SupportFormButtons onClose={() => closeSidePanel()} />
+        <>
+            <SidePanelPaneHeader>
+                <h4 className="flex-1 font-semibold px-2 mb-0">{title}</h4>
+            </SidePanelPaneHeader>
```

Comment:
Not for this PR but should we be allowing people to customize the styling on the title of the side panel? Feels like it could be a `string` prop to maintain consistency